src/index.html

```diff
@@ -7,8 +7,42 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img src="images/logo.png" alt="moyo logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#index.html" class="nav_link is-active">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a href="#index.html" class="nav_link">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a href="#index.html" class="nav_link">Smartphones</a>
+          </li>
+          <li class="nav_item">
+            <a href="#index.html" class="nav_link over"
+            data-qa="hover">Laptops & Computers</a>
+          </li>
+          <li class="nav_item">
```

Comment:
split sibling elements by empty line

Reply:
[checklist:](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)
```suggestion
          </li>
          
          <li class="nav_item">
```